deploy/scripts/src/cri-orange/kbv.ts

```diff
@@ -112,7 +120,7 @@ export function kbv(): void {
   timeGroup(groups[3], () => {
```

Comment:
```suggestion
  timeGroup(groups[4], () => {
```
This one needs updating too